pkg/libmxnet.pkg

```diff
@@ -4,9 +4,9 @@ OPTS = dict(
     name = VAR.fullname,
     description = '''\
 Lightweight, portable deep learning library
- This package provides the core library exposing the MXNet C API.
- .
- It has been built with OpenCV and OpenMP support disabled.
+This package provides the core library exposing the MXNet C API.
+.
```

Comment:
Yea, it's not needed in the `pkg` file.  I'm going to check out a couple of other things too and update accordingly.